crypto/cmp/cmp_local.h

```diff
@@ -48,16 +48,21 @@ struct ossl_cmp_ctx_st {
     void *http_cb_arg; /* allows to store optional argument to cb */
 
     /* server authentication */
-    int unprotectedErrors; /* accept neg. response with no/invalid protection */
-                           /* to cope with broken server */
+    int unprotectedErrors; /*-
+                            * accept neg. response with no/invalid protection
+                            * to cope with broken server
+                            */
```

Comment:
Well, actually that looks clumsy to have a multiline comment hanging over like this.
Could you pleas move those comments before the item, or shorten them a bit?

Reply:
@Akretsch, please just move the multi-line comment without merging it with the comment before.

Note that your current solution:
```
/* server authentication
 *
 * accept neg. response with no/invalid protection
 * to cope with broken server
 */
```
also does not adhere to the format of multi-line comments according to the coding guidelines.